src/coreclr/unwinder/arm64/unwinder.cpp

```diff
@@ -265,16 +269,71 @@ do {
 
 #endif // !defined(DEBUGGER_UNWIND)
 
-//
 // Macros for stripping pointer authentication (PAC) bits.
-//
+#if !defined(DACCESS_COMPILE) && defined(TARGET_ARM64)
```

Comment:
```suggestion
#if !defined(DACCESS_COMPILE) && !defined(FEATURE_CDAC_UNWINDER)`
```
We have two builds of out-of-proc unwinders: DACCESS and CDAC. I think we want to take the offline path for both out-of-proc unwinders. `TARGET_ARM64` should not be needed in the condition once you do enable to the offline path for all out-of-proc unwinders.

(All similar ifdefs in this file should be changed like this.)